packages/python-packages/apistubgentest/setup.py

```diff
@@ -42,6 +42,14 @@
     long_description_content_type='text/markdown',
     license='MIT License',
     author='Microsoft Corporation',
+    install_requires=[
+        "aiohttp",
```

Comment:
hilariously small nit: I'd put a lower bound based on what you reasonably tested.